### ISSUE TITLE
consider-using-enumerate misses case when range uses two arguments

### ISSUE THREAD

**Issue by rogalski** [PyCQA/pylint#1319]
Caught case:
```
for idx in range(len(seq)):
    print(seq[idx])
```

Uncaught case:
```
for idx in range(0, len(seq)):
    print(seq[idx])
```

Pylint version: master

**Comment by PCManticore** [comment]
Sounds good.

**Comment by aaossa** [comment]
Hi, can I work on this? I have a question: Should we caught the second case **only** when the first argument is `0`? If we have something like `range(2, len(seq))` or `range(len(seq) // 2, len(seq))` should be caught? Thanks :raised_hands:

**Comment by rogalski** [comment]
Imho only 0-len case. Enumerate is not subscriptable so those cases are not easily convertable to using enumerate

Good luck!

Dnia 16.04.2017 o godz. 08:41 Antonio Ossa <notifications@github.com> napisał(a):

**Comment by rogalski** [comment]
@aaossa #782 has all relevant steps figured out and seems to be quite friendly for new contributors, but it require changes changes in astroid repository, not pylint code. Other than that - feel free to pick any other non-solved issue. Good luck and thanks for your effort!
